src/models.py

```diff
@@ -1237,6 +1252,40 @@ def _grounded_ranking_bce_forward(self, batch, task, predict):
 
         return out
 
+    def _multiple_choice_reading_comprehension_forward(self, batch, task, predict):
+        ''' Forward call for multiple choice (selecting from a fixed set of answers)
+        reading comprehension (have a supporting paragraph).
+
+        Batch has a tensor of shape (n_questions, n_answers, n_tokens)
+        '''
+        out = {}
+        classifier = self._get_classifier(task)
+        if self.use_bert:
+            inp = batch["para_quest_ans"]
+            ex_embs, ex_mask = self.sent_encoder(inp, task)
+            logits = classifier(ex_embs, ex_mask)
+            out['n_exs'] = inp["bert_wpm_pretokenized"].size(0)
+        else:
```

Comment:
Since BERT is the only supported tokenizer, add a note or assert or something here.

Reply:
Also, how do we get CBOW results in that case?

Reply:
It works with arbitrary tokenization, this branch is just if we're using BERT.